src/vs/platform/terminal/common/capabilities/commandDetectionCapability.ts

```diff
@@ -449,11 +449,6 @@ export class CommandDetectionCapability extends Disposable implements ICommandDe
 		this._logService.debug('CommandDetectionCapability#handleCommandExecuted', this._currentCommand.commandExecutedX, this._currentCommand.commandExecutedMarker?.line);
 	}
 
-	invalidateCurrentCommand(request: ICommandInvalidationRequest): void {
-		this._currentCommand.isInvalid = true;
-		this._onCurrentCommandInvalidated.fire(request);
```

Comment:
Is this event ever used?

Reply:
The event is used but this function is never called (the event is fired elsewhere)